get_commits.sh

```diff
@@ -1,25 +1,146 @@
 #!/bin/bash
 
 set -e
+TAG_UI=${1:-"refs/tags/v2.5.0-rc3"}
+TAG_CONTROLLER=${2:-"refs/tags/v2.5.0-rc1^{}"}
+BRANCH=${3:-"refs/heads/release-v2.5.0"}
```

Comment:
Why is a tag needed for UI and controller instead of branch?

Reply:
I thought that this script is to check the installed build in the ocp and compare it to the upstream commits which would not be affected by the backport if you would have the build with the backports installed in ocp

Reply:
Yeah, that could be an option, but I still don't follow the reasoning behind the tags.
But I'm okay to accept the PR as is if there is some reason.

Reply:
ahhhh for the QE, now I understand and agree with the two options